routes/areas.js

```diff
@@ -10,6 +10,45 @@ Do not think we need a get areas by year because can just
 use the get all areas request then filter based on year on frontend
 */
 
+// get an area by id
+router.get('/area-management', async (req, res) => {
+  try {
+    const areaInfo = await pool.query(`
+      SELECT area.*, site.num_sites, site.site_info, mt_info.num_mts, student_info.num_students
```

Comment:
Do you know if there's a way to also return the date the area was created? If so can you also include that as one of the things returned

Reply:
not possible in postgresql